templates/candidates-single.html

```diff
@@ -58,8 +58,8 @@ <h4 class=>Party</h4>
             {% if leadership_committees %}
             <dl>
               <dt>Leadership PACs</dt>
-              {% for committee in leadership_committees %}
-              <dd><a {% if committee.name_url %}href="{{ committee.name_url }}"{% endif %}>{{ committee.name }}</a></dd>
+              {% for c_id in leadership_committees %}
+              <dd><a {% if leadership_committees[c_id].url %}href="{{ leadership_committees[c_id].url }}"{% endif %}>{{ leadership_committees[c_id].name }}</a></dd>
```

Comment:
I just commented belatedly, you didn't miss anything.

Thanks. I see your point. I think, as we simplify the connection between the templates and the data from the API, we'll just build this URL in the template using the ID. And, worst case scenario, we have an A with no href. I personally don't think that's the biggest deal, could be wrong.